virtual/docker/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>luna@nullrequest.com</email>
+    <description>Primary maintainer</description>
```

Comment:
I'd drop this, it's implied, given the other maintainer has the `proxied` attribute.